File: Palindrome.java
public class Palindrome {
  public static void main(String [] args) {
    String str = args[0].toLowerCase();
    int len = str.length();
    Boolean truthy = true;
    char [] splitStr = new char[len];
    for (int i = 0; i < len; i++) {
      splitStr[i] = str.charAt(i);
    }
    for (int a = 0; a <= splitStr.length - 1; a++) {
      for (int b = splitStr.length - 1; b >= 0; b--) {
        if (splitStr[a] != splitStr[b]) {
          truthy = false;
        } else {
          truthy = true;
        }
      }
    }
    if (truthy) {
      System.out.println(str + " is a palindrome.");
    } else {
      System.out.println(str + " is not a palindrome.");
    }
  }
}

File: palindrome.js
function palindrome (string) {
  var splitStr = string.toLowerCase().split('');
  var truth = [];
  var truthy = true;
  for (var i = 0; i < (splitStr.length - 1); i++) {
    for (var j = (splitStr.length - 1); j > 0; j--) {
      if (splitStr[i] !== splitStr[j]) {
        truthy = false;
      } else {
        truthy = true;
      }
    }
  }
  if (truthy) {
    console.log(string + ' is a palindrome');
  } else {
    console.log(string + ' is not a palindrome');
  }
}


palindrome('Racecar');
palindrome('Michael');
palindrome('Taco cat');
palindrome('Michael DiGuiseppi is awesome');
